packages/rest-typings/src/v1/emojiCustom.ts

```diff
@@ -52,7 +52,7 @@ export const isEmojiCustomList = ajv.compile<emojiCustomList>(emojiCustomListSch
 
 export type EmojiCustomEndpoints = {
 	'/v1/emoji-custom.all': {
-		GET: (params: PaginatedRequest<{ query: string }, 'name'>) => PaginatedResult<{
+		GET: (params: PaginatedRequest<{ query?: string }, 'name'>) => PaginatedResult<{
```

Comment:
it is missing the new `name` query param:
```suggestion
		GET: (params: PaginatedRequest<{ query?: string; name?: string }, 'name'>) => PaginatedResult<{
```